app/views/articles/_actions.html.erb

```diff
@@ -19,7 +19,7 @@
       <%= render partial: "articles/reaction_button",
                  locals: {
                    category: :readinglist,
-                   description: "Saved",
+                   description: "Reading list",
                    image_path: "save.svg",
                    image_active_path: "save-filled.svg"
```

Comment:
This fix for the action buttons is starting to look really good - I did forget one thing with my proposed solution though, and that is setting the aria-label on first load. I think the easiest way to do it is to add an extra property for the aria-label in each action here, e.g.:

```ruby
<%= render partial: "articles/reaction_button",
                 locals: {
                   category: :like,
                   description: "Heart",
                   image_path: "heart.svg",
                   image_active_path: "heart-filled.svg",
                   aria_label: "Like"
                 } %>
```

And then use this in the `_reaction_button.html.erb` (suggestion added below)

Reply:
Indeed, I missed that too.